src/dsp_tools/utils/connection_live.py

```diff
@@ -51,18 +51,15 @@ def login(self, email: str, password: str) -> None:
             password: password of the user
 
         Raises:
-            BaseError: if DSP-API returns no token with the provided user credentials
+            PermanentConnectionError: if DSP-API returns no token with the provided user credentials
         """
         response = self.post(
             route="/v2/authentication",
             data={"email": email, "password": password},
         )
         if not response.get("token"):
-            raise BaseError(
-                f"Error when trying to login with user '{email}' and password '{password} "
-                f"on server '{self.server}'",
-                json_content_of_api_response=json.dumps(response),
-                api_route="/v2/authentication",
+            raise PermanentConnectionError(
+                f"Error when trying to login with user '{email}' and password '{password} on server '{self.server}'"
```

Comment:
don't we log the errors in https://github.com/dasch-swiss/dsp-tools/blob/8296254cfeb54572da2e47788f389246176ecdba/src/dsp_tools/cli/entry_point.py#L68 so would it not log the password here even though we carefully extracted it previously?

Reply:
Good point. I will filter it out here, too.